apps/server/src/modules/board/service/event/user-deleted-event-handler.service.ts

```diff
@@ -0,0 +1,67 @@
+import {
+	DataDeletedEvent,
+	DataDeletionDomainOperationLoggable,
+	DeletionService,
+	DomainDeletionReport,
+	DomainDeletionReportBuilder,
+	DomainName,
+	DomainOperationReportBuilder,
+	OperationType,
+	StatusModel,
+	UserDeletedEvent,
+} from '@modules/deletion';
+import { Injectable } from '@nestjs/common';
+import { EventBus, EventsHandler, IEventHandler } from '@nestjs/cqrs';
+import { BoardExternalReferenceType } from '@shared/domain/domainobject';
+import { EntityId } from '@shared/domain/types';
+import { Logger } from '@src/core/logger';
+import { MediaBoardService } from '../media-board';
+
+@Injectable()
+@EventsHandler(UserDeletedEvent)
+export class UserDeletedEventHandlerService implements DeletionService, IEventHandler<UserDeletedEvent> {
+	constructor(
+		private readonly mediaBoardService: MediaBoardService,
+		private readonly logger: Logger,
+		private readonly eventBus: EventBus
+	) {}
+
+	public async handle({ deletionRequestId, targetRefId }: UserDeletedEvent): Promise<void> {
+		const dataDeleted: DomainDeletionReport = await this.deleteUserData(targetRefId);
+
+		await this.eventBus.publish(new DataDeletedEvent(deletionRequestId, dataDeleted));
```

Comment:
This was something which was commented by alex before: inline creation

Reply:
This was done by DP so I would assume that this is the way they want to have it

Reply:
I don't think so, but I can live with it anyway :P